mergify_engine/context.py

```diff
@@ -52,6 +52,25 @@
 # https://developer.github.com/v4/enum/mergestatestatus/
 
 
+def get_logger(pull):
```

Comment:
Why isn't that a method?

Reply:
It's also used without the context now, I moved it in utils.py